tests/pulser_simulation/test_simulation.py

```diff
@@ -1701,3 +1701,47 @@ def test_initial_state_sim():
             1e-2,
         )
     )
+
+
+def test_amp_sigma_noise(mod_device):
+    reg = Register({"q0": (0, 0), "q1": (10, 10)})
+    seq = Sequence(reg, mod_device)
+    seq.declare_channel("ch0", "rydberg_global")
+    seq.declare_channel("ch1", "raman_local", initial_target="q0")
```

Comment:
The implementation that Antoine suggested would suffer from the bug that two local channels targeting different qubits at the same timestep would not have their noise factors applied correctly. To test for this, it sounds like a good idea to add a second local channel in the test. I will do the same thing on my end.